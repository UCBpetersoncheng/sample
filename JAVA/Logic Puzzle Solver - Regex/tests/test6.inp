John is not the architect.
John is not the teacher.
John is not the warden.
Paul is not the architect.
Paul is not the teacher.
Paul is not the warden.
Mary lives in the yellow house.
Jane lives in the blue house.
Kim is not the architect.
Kim is not the teacher.
What do you know about Kim?