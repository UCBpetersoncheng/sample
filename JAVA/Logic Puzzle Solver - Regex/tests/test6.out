1. John is not the architect.
2. John is not the teacher.
3. John is not the warden.
4. Paul is not the architect.
5. Paul is not the teacher.
6. Paul is not the warden.
7. Mary lives in the yellow house.
8. Jane lives in the blue house.
9. Kim is not the architect.
10. Kim is not the teacher.

Q: What do you know about Kim?
A: Kim is the warden.
